### DISCUSSION TITLE
account access catastrophe (unable to access mailbox for verification code)

### DISCUSSION THREAD

**Discussion by andjb** [community/community#118577] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,
I have another account which I created with a company email, but since leaving the company can no longer access the mailbox to retrieve the verification code.

I'd like to get access to my X years with of personal work (which I foolishly stored under my company email...)

I have the 2fa codes on my mobile, I know the email and username.

If anyone could assist I'd appreciate it!
Thanks in advance

**Comment by ebndev** [comment]
Hi [andjb](https://github.com/andjb)! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
